polaris-icons/icons/Rows2Major.yml

```diff
@@ -0,0 +1,14 @@
+name: Rows 2
+set: major
+description: Used to represent a 2 row layout.
```

Comment:
Will this always only be used to represent only 2 rows, or could there be several since this is a more generic multi-row feature?

Reply:
Technically for our use case there can definitely be more than 2 rows, but I believe we just need the icon in the section list and picker to be a _better_ representation of the section than the generic default icon. Our multicolumn section is also always represented by a 2 column icon even though it will often be used to display more columns than that.

Reply:
I agree that it should be something more generic so it can be used for any multi-row and multi-column situations. But noticed that we already have the column 1, 2, and 3 icons in our library, so I guess it makes sense to have row version of it too.

Reply:
Sounds good to me, I agree it makes the most sense to have parity in names. Folks can of course use it to represent more than 2 rows 💯